modules/foleys_gui_magic/State/foleys_MagicProcessorState.cpp

```diff
@@ -226,6 +226,21 @@ int MagicProcessorState::getLastController() const
     return midiMapper.getLastController();
 }
 
+int MagicProcessorState::getLastMidiChannel() const
```

Comment:
How about exposing the midiMapper instead of cluttering the MagicProcessorState API?

Reply:
Sounds fine to me - I was just following the pattern I found in front of me, really shooting for "minimum perturbation".